taxi/views.py

```diff
@@ -54,23 +63,31 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
 
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
-    paginate_by = 5
     queryset = Car.objects.all().select_related("manufacturer")
```

Comment:
Using .all before prefetch_related is redundant because prefetch_related already retrieves all related objects.